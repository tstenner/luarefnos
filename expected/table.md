   Revision      Date     Change Description
  ---------- ------------ --------------------
     0.5      2022-07-05  Initial Draft
