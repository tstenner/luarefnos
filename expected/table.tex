\begin{longtable}[]{@{}ccl@{}}
\toprule\noalign{}
Revision & Date & Change Description \\
\midrule\noalign{}
\endhead
\bottomrule\noalign{}
\endlastfoot
0.5 & 2022-07-05 & Initial Draft \\
\end{longtable}
